Network/HTTP/ReverseProxy.hs

```diff
@@ -353,18 +354,25 @@ waiProxyToSettings getDest wps' manager req0 sendResponse = do
     case edest of
         Left app -> maybe id timeBound (lpsTimeBound lps) $ app req0 sendResponse
         Right (ProxyDest host port, req, secure) -> tryWebSockets wps host port req sendResponse $ do
-            let req' = def
-                    { HC.method = WAI.requestMethod req
+            let req' =
+#if MIN_VERSION_http_client(0, 5, 0)
+                  HC.defaultRequest
+                    { HC.checkResponse = \_ _ -> return ()
+                    , HC.responseTimeout = maybe HC.responseTimeoutNone (HC.responseTimeoutMicro . (1000 *)) $ lpsTimeBound lps
```

Comment:
I don't think the `1000 *` is correct/necessary here, aren't both values in microseconds?